params/config.go

```diff
@@ -509,6 +511,7 @@ type ChainConfig struct {
 	ArrowGlacierBlock   *big.Int `json:"arrowGlacierBlock,omitempty"`   // Eip-4345 (bomb delay) switch block (nil = no fork, 0 = already activated)
 	ArchimedesBlock     *big.Int `json:"archimedesBlock,omitempty"`     // Archimedes switch block (nil = no fork, 0 = already on archimedes)
 	ShanghaiBlock       *big.Int `json:"shanghaiBlock,omitempty"`       // Shanghai switch block (nil = no fork, 0 = already on shanghai)
+	CancunBlock         *big.Int `json:"cancunBlock,omitempty"`         // Cancun switch block (nil = no fork, 0 = already on cancun)
```

Comment:
We should not do this since we won't really fork with this PR. And this will make it very messy when we actually adopt Cancun changes (new opcodes, etc.). Let's remove the genesis config changes.

Reply:
yeah. removed in [42452b4](https://github.com/scroll-tech/go-ethereum/pull/626/commits/42452b4b1e7c67295418aba71dee688ac782fde0)